### ISSUE TITLE
mapPlot() distortion with mapproj-type projection

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#663]
The sample code (see the test suite item [663](https://github.com/dankelley/oce-issues/tree/master/663))
```R
library(oce)
data(coastlineWorldFine, package="ocedata")
proj <- "mercator"
fill <- 'lightgray'
latlim <- c(43,54)
lonlim <- c(-67,-50)
par(mar=c(2,2,1,1), mfrow=c(2,1))
mapPlot(coastlineWorldFine,latitudelim=latlim, longitudelim=lonlim,
        proj="mercator", fill=fill, grid=c(2,2))
mapPlot(coastlineWorldFine,latitudelim=latlim, longitudelim=lonlim,
        proj="+proj=merc", fill=fill, grid=c(2,2))
```
produces as below. At least to my (mainland) eyes, Newfoundland is distorted in the upper diagram, which uses the mapproj-style projection.  (I think it is using the non-projected projection, just giving degrees of latitude and longitude in a cartesian way.)
![01](https://cloud.githubusercontent.com/assets/99469/8119777/a1fd5910-106e-11e5-9674-a3d71766a4e3.png)

**Comment by dankelley** [comment]
This is fixed now in the ``develop`` branch (commit 54074fc4f51d3d499a3f092c3396f268a531ec27), so I am closing the issue. **HOWEVER** this bug has finally convinced me that dropping mapproj support is a high priority, and I will do that within hours, days, or weeks.

**Comment by dankelley** [comment]
Below is what the sample code now produces, using the ``develop`` branch of oce. Note that there is now a new warning from oce, namely:
```
Warning message:
In mapPlot(coastlineWorldFine, latitudelim = latlim, longitudelim = lonlim,  :
  use PROJ.4 format, e.g. projection="+proj=merc" for Mercator
  (this warning will turn into an error sometime during 2015)
```
![663a](https://cloud.githubusercontent.com/assets/99469/8133349/419ad904-10fe-11e5-9ed6-3e1d7335dbec.png)
